### PR TITLE
Solve issues #236, #239, #240, #241 and #242

### PR DESCRIPTION
**Pull request by DavidBlavid**
Issues: https://github.com/KGQA/leaderboard/issues/236, https://github.com/KGQA/leaderboard/issues/239, https://github.com/KGQA/leaderboard/issues/240, https://github.com/KGQA/leaderboard/issues/241, https://github.com/KGQA/leaderboard/issues/242

### CODE REVIEW

File: wikidata/CronQuestions.md
Comment (xixi019) [inline wikidata/CronQuestions.md:1]: TSQA-> TSQR

File: systems.md
Comment (xixi019) [inline systems.md:1]: TSQA-> TSQR

File: freebase/ComplexWebQuestions.md
Comment (xixi019) [inline freebase/ComplexWebQuestions.md:1]: add dataset Cr-lt-kgqa and its leaderboard mentioned in the paper: https://arxiv.org/pdf/2405.13873

**Review by xixi019 — CHANGES_REQUESTED** [review]

File: wikidata/CronQuestions.md
Comment (DavidBlavid) [reply wikidata/CronQuestions.md:1]: I assume this is a typo, the original authors call their system TSQA. I would leave it as TSQA.
See here: https://arxiv.org/pdf/2203.00255

**Review by xixi019 — APPROVED** [review]

**Pull request merged**